src/formatDate.js

```diff
@@ -51,6 +51,26 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+
+  const arrDate = date.split(fromFormat[3]);
+  const result = [];
+
+  result[toFormat.indexOf('DD')] = arrDate[fromFormat.indexOf('DD')];
+  result[toFormat.indexOf('MM')] = arrDate[fromFormat.indexOf('MM')];
+
+  let year = (fromFormat.indexOf('YYYY') === -1)
+    ? arrDate[fromFormat.indexOf('YY')]
+    : arrDate[fromFormat.indexOf('YYYY')];
+
+  if (toFormat.indexOf('YYYY') === -1) {
```

Comment:
To check if 'YYYY' is present in 'toFormat', use 'includes' method instead of checking for index.